### DISCUSSION TITLE
How do you revive a villager in 1.16.5 when a tombstone didn't exist?

### DISCUSSION THREAD

**Discussion by myriadofdreams** [Luke100000/minecraft-comes-alive#352] [Q&A] [answered]
I have the uuid of the villager too but it seems the revive command is non-existent in this version.

The problem is I didn't have a tombstone so there is nothing to revive. I did a bit of digging and apparently there was a way to revive using the villager's id? Thank you for the quick response btw!

**Comment by CDAGaming** [comment, marked as answer]
(See comment for edits) All features from the current 1.18.2 version of MCA are present in the new revision of 1.16.5, and can be revived with either the scythe or staff of life.

**Reply by CDAGaming** [reply]
As for your edit, yea in modern versions of MCA, it uses the tombstone, but if the villager died and there isn't one, it is gone for good.
